sdk/textanalytics/ai-text-analytics/src/analyzeBatchActionsResult.ts

```diff
@@ -184,12 +183,12 @@ export type PagedAsyncIterableAnalyzeBatchActionsResult = PagedAsyncIterableIter
  */
 export interface PagedAnalyzeBatchActionsResult
   extends PagedAsyncIterableAnalyzeBatchActionsResult {
-  /**
-   * Statistics about the input document batch and how it was processed
-   * by the service. This property will have a value when includeStatistics is set to true
-   * in the client call.
-   */
-  statistics?: TextDocumentBatchStatistics;
+  // /**
```

Comment:
Is this going to be added back in? Can you leave a note about it if so, or just remove it, since it seems simple enough.

Reply:
I left a comment here: https://github.com/Azure/azure-sdk-for-js/blob/998221374cd817a4ac0e2cb369f1e5e2fdf64ef6/sdk/textanalytics/ai-text-analytics/src/lro/analyze/operation.ts#L329.